### DISCUSSION TITLE
Pasting a command option from clipboard doesn't seem to work...?

### DISCUSSION THREAD

**Discussion by petervdn** [discordjs/discord.js#7493] [Q&A] [answered]
I have registered a command, added an option for that command (all with SlashCommandBuilder) which works perfectly fine when i manually type that option/argument. But when i paste the option from clipboard my code does NOT receive that option, even though the command+option in Discord look exactly the same as i would type it.

So this command:
/getUserInfoById 123456

Only seems to work when i actually type the id. When i paste it from clipboard, my interactionCreate listener gets an empty option.

I am retrieving the option value with:
interaction.options.getString('param')

I have tried with string and an integer type option, but they both seem to have the same issue. Commands are sent on Discord client in Chrome OSX.

**Comment by monbrey** [comment, marked as answer]
Unfortunately there's nothing discord.js can do to address this as it relates to how the Discord interface handles input.

https://support.discord.com/hc/en-us/articles/213490807-Where-can-I-submit-Feature-Requests-Feedback-and-Suggestions-

**Comment by petervdn** [comment]
I had a feeling this would be fully on Discord's side. That's a bit of a shame, i think it makes commands useless for me.
